lib/segment/src/vector_storage/query_scorer/mod.rs

```diff
@@ -11,3 +16,14 @@ pub trait QueryScorer<TVector: ?Sized> {
 
     fn score_internal(&self, point_a: PointOffsetType, point_b: PointOffsetType) -> ScoreType;
 }
+
+pub fn score_multi<TMetric: Metric>(a: &MultiDenseVector, b: &MultiDenseVector) -> ScoreType {
+    let mut sum = 0.0;
+    for a in a.iter() {
+        sum += b
+            .iter()
+            .map(|b| TMetric::similarity(a, b))
+            .fold(0.0, |a, b| if a > b { a } else { b });
```

Comment:
I find it a bit confusing to have the accumulator named `a` and `b` like the elements of the vectors in the loop.

Reply:
renamed